radicle-tracker/src/lib.rs

```diff
@@ -0,0 +1,63 @@
+use std::hash;
+
+pub mod thread;
+use thread::*;
+
+pub struct Issue<Author> {
+    pub author: Author,
+    pub title: String,
+    pub thread: Thread<Comment<Author>>,
+    pub meta: (), // TODO(fintan): fill in meta data
+}
+
+impl<Author> Issue<Author> {
+    /// ```
+    /// use radicle_tracker::Issue;
+    ///
+    /// let issue = Issue::new(
+    ///     String::from("Monadic"),
+    ///     String::from("Buggy Boeuf"),
+    ///     String::from("We have bugs in our boeuf"),
```

Comment:
```suggestion
    ///     String::from("We have boeuf in our beef."),
```

Reply:
You don't like french?

Reply:
He didn't like that it was a possibly real life scenario :thinking: